ToDoList-frontend/src/app/bucket/bucket.component.html

```diff
@@ -276,13 +289,26 @@ <h3 class="text-3xl font-semibold">
                 <option>Cancelled</option>
               </select>      
           </div>
-            <div class="mb-6">
-          <label class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Priority</label>
-              <select id="bucketTaskPriority" name="bucketTaskPriority" formControlName="bucketTaskPriority" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
-                  <option>Low</option>
-                  <option>Medium</option>
-                  <option>High</option>
-              </select>
+          <div class="mb-6">
+            <label class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Priority</label>
+            <select id="bucketTaskPriority" name="bucketTaskPriority" formControlName="bucketTaskPriority" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
```

Comment:
Nie za duzo tych klas css? :P Niestety, jesli uzwamy tailwinda, to trzeba sobie zrobic cos ala design system. Czyli po prostu tworzymy wlasny komponent `FormInput`, ktory ma te cssy w sobie, przez co nie musimy ich duplikowac za kazdym razem. Swoja droga ciekawe cwiczenie by to bylo, jakbys sobie cos takie stworzyl. ;)

Reply:
Wyekstraktowałem kilka tasiemców w [https://github.com/kamil-ducz/obj-todolist/commit/f4c4792007822bf5826c849705b092b59677e02d](url)
Zrobię sobie jakąś sesję żeby ogólnie w projekcie wyczyścić jakie się da style - o ile o takie podejście Ci chodziło